tests.js

```diff
@@ -2313,3 +2314,10 @@ QUnit.test("Arity of Primitives", function(t) {
     this.assert_equals('arity "' + proc, arity);
   }.bind(this));
 });
+
+QUnit.test("Sound API", function(t) {
+  //
+  // Sound API
+  //
+  this.assert_equals('beep 220 1', undefined);
```

Comment:
Thank you for adding tests!

Can you add documentation as well, in language.html?